src/google/protobuf/compiler/subprocess.cc

```diff
@@ -335,11 +335,9 @@ void Subprocess::Start(const string& program, SearchMode search_mode) {
 
     // Write directly to STDERR_FILENO to avoid stdio code paths that may do
     // stuff that is unsafe here.
-    int ignored;
```

Comment:
We have internal compilation tools to ensure the return value of this function must not be ignored, thus this redundant code.